src/index.css

```
@@ -0,0 +1,13 @@
+@layer reset {
+  html {
+    font-size: 62.5%;
```

```suggestion
    font-size: 6.25%;
```

폰트 사이즈를 62.5%에서 6.25%로 바꾸는건 어떠세요?
1px -> 1rem으로 접근해서 부가적인 계산 없이 쉽게 쓸 수 있을 것 같습니다.

---

62.5%로 정했다고 해서 정확히 1rem이 10px인건 아니에요! 소수점이 없을수록 숫자가 좀 더 정확해지기 때문에 `62.5%`를 사용하는 것이 더 좋을 것 같습니다 : D
관련 글 [링크](https://stackoverflow.com/questions/47923397/css-fonts-rem-trick-62-5-or-6-25) 해둘게요!